coap-core/src/main/java/com/mbed/coap/server/internal/CoapUdpMessaging.java

```diff
@@ -137,9 +165,6 @@ public ScheduledExecutorService getScheduledExecutor() {
 
     @Override
     public synchronized void start(CoapRequestHandler coapRequestHandler) throws IOException, IllegalStateException {
-        if (duplicationDetector != null) {
-            duplicationDetector.start();
```

Comment:
Shouldn't the cache start be called here? In case of the default one. I don't see its start called anywhere else than from tests.

Reply:
I think the starts needs to be in the interface for the cache.

Reply:
I fixed it.